web/site/js/vis/PixiVisualizer.js

```diff
@@ -478,19 +582,42 @@ export default class PixiVisualizer extends VObject {
     }
 
     visAddRouterTable(nodeId, extaddr) {
-        this.nodes[nodeId].addRouterTable(extaddr)
+        this.nodes[nodeId].addRouterTable(extaddr);
+        this.logNode(nodeId, `Established Router link to ${this.formatExtAddrPretty(extaddr)}`)
     }
 
     visRemoveRouterTable(nodeId, extaddr) {
-        this.nodes[nodeId].removeRouterTable(extaddr)
+        this.nodes[nodeId].removeRouterTable(extaddr);
+        this.logNode(nodeId, `Dropped Router link to ${this.formatExtAddrPretty(extaddr)}`)
     }
 
     visAddChildTable(nodeId, extaddr) {
-        this.nodes[nodeId].addChildTable(extaddr)
+        this.nodes[nodeId].addChildTable(extaddr);
+        this.logNode(nodeId, `Child ${this.formatExtAddrPretty(extaddr)} attached`)
     }
 
     visRemoveChildTable(nodeId, extaddr) {
-        this.nodes[nodeId].removeChildTable(extaddr)
+        this.nodes[nodeId].removeChildTable(extaddr);
+        this.logNode(nodeId, `Child ${this.formatExtAddrPretty(extaddr)} detached`)
```

Comment:
Can we use a similar format as to the router link events? E.g. `this.logNode(nodeId, 'Detached child link to ${this.formatExtAddrPretty(extaddr)}')`

Reply:
Changed to `Child table removed: Node X(...)`